src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -89,15 +89,19 @@ export default class MxGraphRenderer {
     if (bpmnElement instanceof ShapeBpmnEvent) {
       styleValues.set(StyleConstant.BPMN_STYLE_EVENT_KIND, bpmnElement.eventKind);
     }
-
-    if (bpmnCell instanceof Shape && labelBounds) {
-      // arbitrarily increase width to relax too small bounds (for instance for reference diagrams from miwg-test-suite)
-      styleValues.set(this.mxConstants.STYLE_LABEL_WIDTH, labelBounds.width + 1);
-      // align settings
-      styleValues.set(this.mxConstants.STYLE_VERTICAL_ALIGN, this.mxConstants.ALIGN_TOP);
-      styleValues.set(this.mxConstants.STYLE_ALIGN, this.mxConstants.ALIGN_CENTER);
-      styleValues.set(this.mxConstants.STYLE_LABEL_POSITION, this.mxConstants.ALIGN_TOP);
-      styleValues.set(this.mxConstants.STYLE_VERTICAL_LABEL_POSITION, this.mxConstants.ALIGN_LEFT);
+    if (labelBounds) {
+      if (bpmnCell instanceof Shape) {
+        // arbitrarily increase width to relax too small bounds (for instance for reference diagrams from miwg-test-suite)
+        styleValues.set(this.mxConstants.STYLE_LABEL_WIDTH, labelBounds.width + 1);
+        // align settings
+        styleValues.set(this.mxConstants.STYLE_VERTICAL_ALIGN, this.mxConstants.ALIGN_TOP);
+        styleValues.set(this.mxConstants.STYLE_ALIGN, this.mxConstants.ALIGN_CENTER);
+        styleValues.set(this.mxConstants.STYLE_LABEL_POSITION, this.mxConstants.ALIGN_TOP);
+        styleValues.set(this.mxConstants.STYLE_VERTICAL_LABEL_POSITION, this.mxConstants.ALIGN_LEFT);
+      } else {
+        styleValues.set(this.mxConstants.STYLE_VERTICAL_ALIGN, this.mxConstants.ALIGN_TOP);
+        styleValues.set(this.mxConstants.STYLE_ALIGN, this.mxConstants.ALIGN_CENTER);
+      }
```

Comment:
As it's the same value as for Shape, we can put above the 'if'.

```suggestion
        styleValues.set(this.mxConstants.STYLE_VERTICAL_ALIGN, this.mxConstants.ALIGN_TOP);
        styleValues.set(this.mxConstants.STYLE_ALIGN, this.mxConstants.ALIGN_CENTER);
        
      if (bpmnCell instanceof Shape) {
        // arbitrarily increase width to relax too small bounds (for instance for reference diagrams from miwg-test-suite)
        styleValues.set(this.mxConstants.STYLE_LABEL_WIDTH, labelBounds.width + 1);
        // align settings
        styleValues.set(this.mxConstants.STYLE_LABEL_POSITION, this.mxConstants.ALIGN_TOP);
        styleValues.set(this.mxConstants.STYLE_VERTICAL_LABEL_POSITION, this.mxConstants.ALIGN_LEFT);
      }
```

Reply:
nice, in addition, this limits code changes

Reply:
Managed by 6bd0db2